index.html

```diff
@@ -75,6 +75,330 @@ <h4>
+              <li><label>End Dev</label></li>
+              <li>
+                <img src="assets/images/icons/Counter.png" 
+                  alt="circle"/>
+              </li>
+              <li><label>2015</label></li>
+            </ul>
+          </div>
+          <div>
+            <p class="project-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required.
+            </p>
+          </div>
+          <ul class="project-button-group">
+            <li>
+              <button type="button" onclick="return;">
+                HTML
+              </button>
+            </li>
+            <li>
+              <button type="button" onclick="return;">
+                CSS
+              </button>
+            </li>
+            <li>
+              <button type="button" onclick="return;">
+                JavaScript
+              </button>
+            </li>
+          </ul>
+          <div>
+            <button class="project-button" type="button"
+              onclick="return;">
+              See project
+            </button>
+          </div>
+        </div>
+        <div class="flex-container card">
+          <img class="project-img" 
+            src="assets/images/samples/Snapshoot Portfolio(4).png" 
+            alt="Portfolio 1" />
+          <h2 class="project-title">
+            Multi-Post Stories
+          </h2>
+          <div>
+            <ul class="flex-list project-date-list">
+              <li><h5>CANOPY</h5></li>
+              <li>
+                <img src="assets/images/icons/Counter.png" 
+                  alt="circle"/>
+              </li>
+              <li><label>End Dev</label></li>
+              <li>
+                <img src="assets/images/icons/Counter.png" 
+                  alt="circle"/>
+              </li>
+              <li><label>2015</label></li>
+            </ul>
+          </div>
+          <div>
+            <p class="project-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required.
+            </p>
+          </div>
+          <ul class="project-button-group">
+            <li>
+              <button type="button" onclick="return;">
+                HTML
+              </button>
+            </li>
+            <li>
+              <button type="button" onclick="return;">
+                CSS
+              </button>
+            </li>
+            <li>
+              <button type="button" onclick="return;">
+                JavaScript
+              </button>
+            </li>
+          </ul>
+          <div>
+            <button class="project-button" type="button"
+              onclick="return;">
+              See project
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about">
+      <div class="about-background flex-container">
+        <h2>
+          About Myself
+        </h2>
```

Comment:
Figma
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/78251927/15fb30fb-8a22-408b-9f4a-6347cb06d985)

Your design
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/78251927/9c021a39-6fbc-4bd2-b1f0-f90c57a77c96)

To make our design as similar to the guideline as possible using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). kindly add the border-radius as indicated above. 💪